src/components/TransactionFlow/TxReceiptTotals.tsx

```diff
@@ -2,14 +2,23 @@ import React from 'react';
 
 import styled from 'styled-components';
 
-import { Amount, AssetIcon } from '@components';
-import { default as Icon } from '@components/Icon';
-import { getFiat } from '@config/fiats';
-import { COLORS, SPACING } from '@theme';
+import { Amount } from '@components';
+import Icon from '@components/Icon';
+import { getFiat } from '@config';
+import { BREAK_POINTS, SPACING } from '@theme';
 import translate from '@translations';
 import { ExtendedAsset, ISettings } from '@types';
 import { bigify, convertToFiat, fromWei, totalTxFeeToWei, Wei } from '@utils';
 
+const SIcon = styled(Icon)`
+  width: 30px;
+  height: 30px;
+  margin-right: ${SPACING.SM};
+  vertical-align: middle;
+  @media (max-width: ${BREAK_POINTS.SCREEN_SM}) {
+    display: none;
```

Comment:
What is the reasoning behind hiding them on mobile?

Reply:
Design hides the asset icon on mobile. https://www.figma.com/file/BY0SWc75teEUZzws8JdgLMpy/MyCrypto-Web-App-Final-Designs?node-id=701%3A1523. Most likely due to save real estate and avoid wrapping. 

I find that the asset icon is more important than the title icon, so I inverted the rule and hide the latter.